### DISCUSSION TITLE
How to know the version of the application contained in the docker image before installation

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#4834] [Questions] [answered]
I don't want to rush into an update.

**Comment by szaimen** [comment]
Hi, you can simply look at https://github.com/nextcloud/all-in-one/releases

**Comment by ghost** [comment]
https://hub.docker.com/r/nextcloud/all-in-one
This is what I really want to ask because I don't know the relationship between the image version on docker hub and the github release。

**Reply by szaimen** [reply, marked as answer]
Beta releases are pushed to the `beta` tag on docker hub, stable releases are pushed to thr `latest` tag. I hope this clarifies things.

**Comment by ghost** [comment]
Oh, my intention was just to know the version of nextcloud when I pull the image image. But the information about the nextcloud version can only be seen in github.
Can I reconcile by publication time and tags? Or do you have a better way?
For example, I guess the following one corresponds, because their labels and times are the same.
https://hub.docker.com/layers/nextcloud/all-in-one/beta/images/sha256-b4f1e2e4a12b34db6e20c16202a6e0b096841a801a8b603d81171de87d5b22f0?context=explore
https://github.com/nextcloud/all-in-one/releases/tag/v9.0.1
